client/client.go

```diff
@@ -611,8 +621,9 @@ var _ internal.NamespaceClient = NamespaceClient(nil)
 // User had Activity.RecordHeartbeat(ctx, "my-heartbeat") and then got response from calling Client.DescribeWorkflowExecution.
 // The response contains binary field PendingActivityInfo.HeartbeatDetails,
 // which can be decoded by using:
-//   var result string // This need to be same type as the one passed to RecordHeartbeat
-//   NewValue(data).Get(&result)
+//
+//	var result string // This need to be same type as the one passed to RecordHeartbeat
+//	NewValue(data).Get(&result)
```

Comment:
Aren't these ones (and in a number of other places) supposed to be indented since they're code blocks? Won't this dedent not render them that way?

Reply:
It's hard to see but the "v" in `var` is under the "h" in `which` above. They're tab indented so they do end up rendering properly.

![image](https://user-images.githubusercontent.com/275947/207115355-80271bc7-fcd9-4677-b027-d22e481a8546.png)